packages/apps/job-launcher/server/docker-compose.yml

```diff
@@ -38,7 +38,7 @@ services:
         condition: service_healthy
     entrypoint: >
       /bin/sh -c "
-      /usr/bin/mc config host add myminio http://minio:9000  access-key secret-key;
+      /usr/bin/mc config host add myminio http://minio:9000 minioadmin minioadmin;
```

Comment:
These are default creds and below cmd creates bucket, so now minio works properly out of the box

Reply:
wasn't this working before?

Reply:
@portuu3 by default - no. In order to make it working you have to either:
- start minio, go to console, create a service account with this access/secret key
- or have a config file for minio to start container with access/secret key defined

I think it's easier to use default credentials from minio itself, esp taking into account it's only for local development